index.html

```diff
@@ -30,6 +30,25 @@
 -->
 
 <div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <div class="col-md-1">
+      </div>
+      <div class="col-md-10 lead text-justify">
+        Interledger is an open protocol suite for sending payments across different <em>ledgers</em>.
```

Comment:
I was trying to reduce the repetition of the word "payment" and introduce the core concept. I hope it's clear from the number of times ledger is used interchangeably with payment network and the typing animation above, but I'm curious to hear what others think about that